server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -221,12 +217,12 @@ export function configureWebSocketServices(
             try {
                 await tenantManager.verifyToken(claims.tenantId, token);
             } catch (err) {
-                // eslint-disable-next-line prefer-promise-reject-errors
-                return Promise.reject({
-                    // if we don't understand the error, be lenient and allow retry
-                    code: err?.response?.status ?? 401,
-                    message: err?.response?.data ?? "Invalid token",
-                });
+                if (typeof (err as NetworkError | undefined)?.code === "number" &&
```

Comment:
Consider writing a [type guard](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates), that's basically what you're doing, but it would isolate all the `as` stuff out of this code.